src/client/Microsoft.Identity.Client/Platforms/iOS/iOSTokenCacheAccessor.cs

```diff
@@ -222,6 +240,27 @@ private SecStatusCode Save(IiOSKey key, string payload)
             return secStatusCode;
         }
 
+        internal SecStatusCode Save(string clientIdAsKey, string applicationToken)
```

Comment:
SaveBrokerApplicationToken

Reply:
i was just following the other methods here, but i also agree it makes sense to specify it. thx.

---
In reply to: [339124173](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1475#discussion_r339124173) [](ancestors = 339124173)